src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -3554,6 +3554,327 @@ class D {  (int, bool) _field; }";
             compilation.VerifyDiagnostics();
         }
 
+        [ConditionalFact(typeof(MonoOrCoreClrOnly), Reason = "Desktop CLR displays argument exceptions differently")]
```

Comment:
Did you consider adding a helper of the style 

```csharp
VerifyDiagonsticArgumentException(...) {
#if NETCOREAPP
...
#else
...
}
```

That would allow all these tests to run on .NET Framework and .NET Core.

Reply:
I copied this ConiditionalFact with the reason from other tests for argument exceptions. But will add a helper.